### DISCUSSION TITLE
모니터링 관련 이미지 증빙자료 입니다.

### DISCUSSION THREAD

**Discussion by youngh0** [woowacourse-teams/2023-stamp-crush#552] [General]
<img width="1865" alt="스크린샷 2023-08-17 오후 4 04 30" src="https://github.com/woowacourse-teams/2023-stamp-crush/assets/62167801/df46f3e4-40ab-4eae-8af8-361d0ba31cab">
<img width="1866" alt="스크린샷 2023-08-17 오후 4 04 03" src="https://github.com/woowacourse-teams/2023-stamp-crush/assets/62167801/9f715098-59ea-4c4f-b487-b35f1db7431a">

**Comment by youngh0** [comment]
<img width="1394" alt="스크린샷 2023-08-17 오후 4 03 13" src="https://github.com/woowacourse-teams/2023-stamp-crush/assets/62167801/fad36b72-276f-4040-b459-c19dbd255c51">

**Comment by gitchannn** [comment]
올.... 레오 수고했슴니당
